models/app.models.js

```diff
@@ -117,3 +117,37 @@ exports.updateArticleVotes = async (id, amt) => {
 		return results[1].rows[0];
 	}
 };
+
+exports.fetchComment = async id => {
+	const results = await db.query(
+		`
+    SELECT * FROM comments
+    WHERE comment_id = $1;
+  `,
+		[id]
+	);
+
+	if (results.rowCount === 0) {
+		return Promise.reject({ code: 404 });
+	} else {
+		return results;
+	}
+};
+
+exports.deleteComment = async id => {
+	const comment = this.fetchComment(id);
```

Comment:
I'm not too sure why you need to fetch the comment here. If you add a RETURNING * to your delete query, you can check whether an entry was actually deleted by the length of the result.rows array you get back. It would save making two calls to achieve the same result